spec/supabase_js_v2_legacy.yml

```diff
@@ -640,281 +640,1199 @@ pages:
     title: 'Fetch data: select()'
     $ref: '@supabase/postgrest-js.PostgrestQueryBuilder.select'
     notes: |
-      - By default, Supabase projects will return a maximum of 1,000 rows. This setting can be changed in Project API Settings. It's recommended that you keep it low to limit the payload size of accidental or malicious requests. You can use `range()` queries to paginate through your data.
-      - `select()` can be combined with [Modifiers](/docs/reference/javascript/using-modifiers)
-      - `select()` can be combined with [Filters](/docs/reference/javascript/using-filters)
-      - If using the Supabase hosted platform `apikey` is technically a reserved keyword, since the API gateway will pluck it out for authentication. [It should be avoided as a column name](https://github.com/supabase/supabase/issues/5465).
+      - By default, Supabase projects return a maximum of 1,000 rows. This setting can be changed in your project's [API settings](https://app.supabase.com/project/_/settings/api). It's recommended that you keep it low to limit the payload size of accidental or malicious requests. You can use `range()` queries to paginate through your data.
+      - `select()` can be combined with [Filters](/docs/reference/javascript/next/using-filters)
+      - `select()` can be combined with [Modifiers](/docs/reference/javascript/next/using-modifiers)
+      - `apikey` is a reserved keyword if you're using the [Supabase Platform](/docs/guides/hosting/platform) and should be avoided as a column name](https://github.com/supabase/supabase/issues/5465).
     examples:
       - name: Getting your data
+        description: |
+          <Tabs>
```

Comment:
@soedirgo sorry to be a PITA, but can we:
- List JavaScript as the first tab since this is in the supabase-js ref docs
- Use `SQL` instead of `Schema` as the tab title to match the rest of the docs
- Make sure to sync these tabs (really helpful for pages with multiple tabs)

Unfortunately, tab syncing is pretty clunky and manual but it looks like this (if you haven't worked with it before):

```
<Tabs
  groupId="js-or-sql"
  defaultValue="js"
  values={[
    {label: 'JavaScript', value: 'js'},
    {label: 'SQL', value: 'sql'},
    {label: 'Result', value: 'result'},
  ]}>

<TabItem value="js">
...
</TabItem>

<TabItem value="sql">
...
</TabItem>

<TabItem value="result">
...
</TabItem>

</Tabs>
```

Note that `groupId` has to be the same value across all the tabs you want to sync and `value` is also case-sensitive.

Reply:
> List JavaScript as the first tab since this is in the supabase-js ref docs

Can do!

> Use SQL instead of Schema as the tab title to match the rest of the docs

Hmm not sure if these are the same - e.g. in [this section](https://supabase.com/docs/guides/database/functions#simple-functions) the `SQL` tab is mimicking the query in the `JS` and `Dart` tab. But in this case the `Schema` section is showing what's actually in the db, rather than the query itself.

> Make sure to sync these tabs (really helpful for pages with multiple tabs)

Maybe syncing tabs are a better fit for multi-language pages? As a user I'd want to switch tabs independently since each snippet is self-contained - c.f. Stripe docs where users might be only interested in one language at a time, so syncing tabs makes sense.

Reply:
Will leave the decisions above up to you - personally I just want the examples to be up there somewhere 😄 I'll merge this for now and address these in a follow-up PR